docs/guides/validation/how_to_validate_data_by_running_a_checkpoint.md

```diff
@@ -0,0 +1,84 @@
+---
+title: How to validate your data using a Checkpoint
+---
+
+import Prerequisites from '../../guides/connecting_to_your_data/components/prerequisites.jsx';
+
+This guide will help you validate your data by running a Checkpoint.
+
+
+<Prerequisites>
+
+- Configured a [Data Context](../../tutorials/getting_started/initialize_a_data_context.md).
+- Configured an [Expectations Suite](../../tutorials/getting_started/create_your_first_expectations.md).
+- Configured a [Checkpoint](./checkpoints/how_to_create_a_new_checkpoint)
+
+</Prerequisites>
+
+Steps
+-----
+
+1. You can update your existing checkpoint either using a text editor or by executing the following command:
```

Comment:
Could we please capitalize checkpoint just for consistency?

Reply:
@cdkini Nice catch -- fixed.  Thank you!